### DISCUSSION TITLE
Disable VPN temporarily

### DISCUSSION THREAD

**Discussion by ArneNaessens** [haugene/docker-transmission-openvpn#2468] [Q&A]
Hello,

I was wondering if i could "disable" the VPN function.
For debugging perposes i would like to not have a vpn connected to this container.

I have tried commenting out every VPN related line in the docker-compose file but that resulted in my container not starting since it couldn't find vpn files.

Is there something for this?

**Comment by ilike2burnthing** [comment]
https://github.com/haugene/docker-transmission-openvpn/discussions/2143
